templates/index.html

```diff
@@ -3,191 +3,183 @@
   <head>
     <meta charset="utf-8">
     <title>IATI Data Store CSV Query Builder (Alpha Version)</title>
-    <link href="style.css" type="text/css" rel="stylesheet">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
+    <link href="custom.css" type="text/css" rel="stylesheet">
   </head>
-  <body dir="ltr" class="ss-base-body">
-    <div itemscope itemtype="http://schema.org/CreativeWork/FormObject">
-      <div class="ss-form-container">
-        <div class="ss-top-of-page">
-          <div class="ss-form-heading">
-            <h1 class="ss-form-title" dir="ltr">IATI Data Store CSV Query Builder (Alpha Version)</h1>
-            <p class="guide">Please read the <a href="http://datastore.iatistandard.org/docs/user-guide/">User Guide</a></p>
-            <hr class="ss-email-break" style="display:none;">
-
-            {% if api_link %}
-            <div class="url">
+  <body dir="ltr">
+    <div class="container">
+      <div class="header row">
+        <div class="col">
+          <h1 dir="ltr">IATI Data Store CSV Query Builder (Alpha Version)</h1>
+          <p>Please read the <a href="http://datastore.iatistandard.org/docs/user-guide/">user guide</a></p>
+        </div>
+      </div><!-- end class="header" -->
+
+      <div class="query-builder">
+        <div class="row"><!-- result -->
+          {% if api_link %}
```

Comment:
Should there be a matching `endif`?

Reply:
Just checked this and the `endif` for this statement is on line 37.  Have fixed the indentation for a nested `if` statement in 0219736 (which previously made it harder to see the `if` / `elseif` / `endif` control flow).